.github/workflows/archived/coding-v3.3.1-rc.yml

```
@@ -0,0 +1,137 @@
+      - name: Set up Node.js
+        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+        with:
+          cache: npm
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: npm ci
+        env:
+          TEST: true
+      - name: Check code
+        run: ls -all
+      # - name: Lint code  
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)